Userland/Libraries/LibCompress/BrotliDictionary.cpp

```diff
@@ -0,0 +1,7902 @@
+    0x77, 0x69, 0x64, 0x74, 0x68, 0x3d, 0x22, 0x32, 0x6c, 0x61, 0x7a, 0x79, 0x6c, 0x6f, 0x61, 0x64,
+    0x6e, 0x6f, 0x76, 0x65, 0x6d, 0x62, 0x65, 0x72, 0x75, 0x73, 0x65, 0x64, 0x20, 0x69, 0x6e, 0x20,
+    0x68, 0x65, 0x69, 0x67, 0x68, 0x74, 0x3d, 0x22, 0x63, 0x72, 0x69, 0x70, 0x74, 0x22, 0x3e, 0x0a,
+    0x26, 0x6e, 0x62, 0x73, 0x70, 0x3b, 0x3c, 0x2f, 0x3c, 0x74, 0x72, 0x3e, 0x3c, 0x74, 0x64, 0x20,
+    0x68, 0x65, 0x69, 0x67, 0x68, 0x74, 0x3a, 0x32, 0x2f, 0x70, 0x72, 0x6f, 0x64, 0x75, 0x63, 0x74,
+    0x63, 0x6f, 0x75, 0x6e, 0x74, 0x72, 0x79, 0x20, 0x69, 0x6e, 0x63, 0x6c, 0x75, 0x64, 0x65, 0x20,
+    0x66, 0x6f, 0x6f, 0x74, 0x65, 0x72, 0x22, 0x20, 0x26, 0x6c, 0x74, 0x3b, 0x21, 0x2d, 0x2d, 0x20,
+    0x74, 0x69, 0x74, 0x6c, 0x65, 0x22, 0x3e, 0x3c, 0x2f, 0x6a, 0x71, 0x75, 0x65, 0x72, 0x79, 0x2e,
+    0x3c, 0x2f, 0x66, 0x6f, 0x72, 0x6d, 0x3e, 0x0a, 0x28, 0xe7, 0xae, 0x80, 0xe4, 0xbd, 0x93, 0x29,
+    0x28, 0xe7, 0xb9, 0x81, 0xe9, 0xab, 0x94, 0x29, 0x68, 0x72, 0x76, 0x61, 0x74, 0x73, 0x6b, 0x69,
+    0x69, 0x74, 0x61, 0x6c, 0x69, 0x61, 0x6e, 0x6f, 0x72, 0x6f, 0x6d, 0xc3, 0xa2, 0x6e, 0xc4, 0x83,
+    0x74, 0xc3, 0xbc, 0x72, 0x6b, 0xc3, 0xa7, 0x65, 0xd8, 0xa7, 0xd8, 0xb1, 0xd8, 0xaf, 0xd9, 0x88,
+    0x74, 0x61, 0x6d, 0x62, 0x69, 0xc3, 0xa9, 0x6e, 0x6e, 0x6f, 0x74, 0x69, 0x63, 0x69, 0x61, 0x73,
+    0x6d, 0x65, 0x6e, 0x73, 0x61, 0x6a, 0x65, 0x73, 0x70, 0x65, 0x72, 0x73, 0x6f, 0x6e, 0x61, 0x73,
+    0x64, 0x65, 0x72, 0x65, 0x63, 0x68, 0x6f, 0x73, 0x6e, 0x61, 0x63, 0x69, 0x6f, 0x6e, 0x61, 0x6c,
+    0x73, 0x65, 0x72, 0x76, 0x69, 0x63, 0x69, 0x6f, 0x63, 0x6f, 0x6e, 0x74, 0x61, 0x63, 0x74, 0x6f,
+    0x75, 0x73, 0x75, 0x61, 0x72, 0x69, 0x6f, 0x73, 0x70, 0x72, 0x6f, 0x67, 0x72, 0x61, 0x6d, 0x61,
+    0x67, 0x6f, 0x62, 0x69, 0x65, 0x72, 0x6e, 0x6f, 0x65, 0x6d, 0x70, 0x72, 0x65, 0x73, 0x61, 0x73,
+    0x61, 0x6e, 0x75, 0x6e, 0x63, 0x69, 0x6f, 0x73, 0x76, 0x61, 0x6c, 0x65, 0x6e, 0x63, 0x69, 0x61,
+    0x63, 0x6f, 0x6c, 0x6f, 0x6d, 0x62, 0x69, 0x61, 0x64, 0x65, 0x73, 0x70, 0x75, 0xc3, 0xa9, 0x73,
+    0x64, 0x65, 0x70, 0x6f, 0x72, 0x74, 0x65, 0x73, 0x70, 0x72, 0x6f, 0x79, 0x65, 0x63, 0x74, 0x6f,
+    0x70, 0x72, 0x6f, 0x64, 0x75, 0x63, 0x74, 0x6f, 0x70, 0xc3, 0xba, 0x62, 0x6c, 0x69, 0x63, 0x6f,
+    0x6e, 0x6f, 0x73, 0x6f, 0x74, 0x72, 0x6f, 0x73, 0x68, 0x69, 0x73, 0x74, 0x6f, 0x72, 0x69, 0x61,
+    0x70, 0x72, 0x65, 0x73, 0x65, 0x6e, 0x74, 0x65, 0x6d, 0x69, 0x6c, 0x6c, 0x6f, 0x6e, 0x65, 0x73,
+    0x6d, 0x65, 0x64, 0x69, 0x61, 0x6e, 0x74, 0x65, 0x70, 0x72, 0x65, 0x67, 0x75, 0x6e, 0x74, 0x61,
+    0x61, 0x6e, 0x74, 0x65, 0x72, 0x69, 0x6f, 0x72, 0x72, 0x65, 0x63, 0x75, 0x72, 0x73, 0x6f, 0x73,
+    0x70, 0x72, 0x6f, 0x62, 0x6c, 0x65, 0x6d, 0x61, 0x73, 0x61, 0x6e, 0x74, 0x69, 0x61, 0x67, 0x6f,
+    0x6e, 0x75, 0x65, 0x73, 0x74, 0x72, 0x6f, 0x73, 0x6f, 0x70, 0x69, 0x6e, 0x69, 0xc3, 0xb3, 0x6e,
+    0x69, 0x6d, 0x70, 0x72, 0x69, 0x6d, 0x69, 0x72, 0x6d, 0x69, 0x65, 0x6e, 0x74
```

Comment:
This can fail:
[crash-8fe6d7ff55d1c699f3e8f979201e32e90389da43.txt](https://github.com/SerenityOS/serenity/files/8422021/crash-8fe6d7ff55d1c699f3e8f979201e32e90389da43.txt)